frontend/public/manifest.json

```diff
@@ -0,0 +1,15 @@
+{
+  "short_name": "React App",
+  "name": "Create React App Sample",
```

Comment:
These should probably be made specific to Mimic, right?

Reply:
Haha true. Fixed! We also should have a favicon/logo

Reply:
Hmm, might be a bit late, but we could ask design for one 😅